plugins/Toolbox/resources/qml/ToolboxDetailPage.qml

```diff
@@ -33,6 +33,7 @@ Item
             height: UM.Theme.getSize("toolbox_thumbnail_medium").height
             fillMode: Image.PreserveAspectFit
             source: details.icon_url || "../images/logobot.svg"
+            mipmap: true
```

Comment:
Nifty!

Reply:
It does require a slightly newer qtquick (2.3 vs 2.2), but that seems to be available with pyqt 5.8 (and newer)

Reply:
Yes we use 2.3 for some other things as well as I recall.